### DISCUSSION TITLE
WARNING: Display power supply voltage

### DISCUSSION THREAD

**Discussion by Bodmer** [Bodmer/TFT_eSPI#1721] [Alerts]
Some displays do NOT have a 5V to 3.3V regulator fitted, so in that case you must ONLY connect 3V3 (3.3 Volts) to the display.

Check the display vendors web page and look for the power voltage needs.

A display that accepts only 3.3V may be destroyed when connecting 5V. Most survive for a while due to voltage drops but operation will be unreliable and the display may overheat.

**Comment by griffinmt** [comment]
A very good warning. Wish I read it earlier. I have a GC9A01 display that just had that problem!!
